### ISSUE TITLE
4.1.3 DID Resolution

### ISSUE THREAD

**Issue by bluesteens** [Open-Credentialing-Initiative/Digital-Wallet-Conformance-Criteria#16]
the para says,

> Digital Wallet Providers SHALL integrate a [DID] resolver **service** for resolving [DID] documents. The [DID] resolver SHALL support all [DID] methods endorsed by the OCI as defined in Table 3 in section 3.1.2. It SHALL be a trusted open source [DID] resolver with broad community participation (e.g. DIF Universal Resolver or DIDKit) vetted against the non-functional conformance criteria outlined in this document. Wallet providers SHALL NOT use public instances of resolvers, instead using their own implementations.

It is not required to use a 3rd party SERVICE to enable DID resolution. Thus I suggest to rephrase the para as follows (2 changes in bold text),

> Digital Wallet Providers SHALL integrate a [DID] resolver **functionality** for resolving [DID] documents. The [DID] resolver SHALL support all [DID] methods endorsed by the OCI as defined in Table 3 in section 3.1.2. It SHALL be a trusted open source [DID] resolver with broad community participation (e.g. DIF Universal Resolver, DIDKit, **Veramo**) vetted against the non-functional conformance criteria outlined in this document. Wallet providers SHALL NOT use public instances of resolvers, instead using their own implementations.

This wording has the reader interpret the para in terms of functionality however this is achieved, by using a 3rd party or an own implementation, either one required to meet the specified trust/quality criteria. It also adds Veramo as another example that facilitates the implementation of an own solution (vs. Uniresolver being a 3rd party service).

**Comment by bluesteens** [comment]
**Triage:**

- [ ] Is Issue appropriate for OCI Architecture
- [ ] Assign **Size**
- [ ] Assign **Priority**
- [ ] Assign **Label** (if needed)
- [ ] OCI affected Artifacts Identified
- [ ] Assign **Triage - Artifact Version Target** (v x.x.x Milestone)
- [ ] Assign **Triage - Interop Profile Version Target** (v x.x.x Milestone)
- [ ] Create sub-project (if needed)

**Comment by alexcolganLD** [comment]
Agreed, though change "integrate a [DID] resolver functionality" --> integrate [DID] resolver functionality"

**Comment by bluesteens** [comment]
July, 27: to be released in Digital Wallet Criteria v3.3.0,  interop profile v3.2.0
